website/src/pages/index.tsx

```diff
@@ -222,4 +228,4 @@ export default ((props: any) => (
             </Jumbotron>
         </div>
     </Layout>
-)) as React.FunctionComponent<any>
+)) as React.FunctionComponent < any >
```

Comment:
```suggestion
)) as React.FunctionComponent<any>
```

This should be reverted.